tests/test_config_parser.py

```diff
@@ -121,12 +121,16 @@ def grep_ir_cmd(self, token: Token) -> None:
         self.config.grep_ir_strs.append(token.strip())
 
 
-def parse_file(path: Path) -> TestConfig:
+def parse_file(path: Path) -> TestConfig | None:
     with path.open(encoding="utf-8") as file:
         lines = [
             line.removeprefix("///").strip() for line in file if line.startswith("///")
         ]
 
+    if not lines:
+        # This is not a test file.
+        return None
```

Comment:
Hmm, this will still crash if the syntax of the testing comments is wrong right?

This is still better than before tho

Reply:
Yes, which is the right thing to do I believe (we're not testing the testing commands). I've improved this now to print the file path and the message, but not the whole stack trace.